### PR TITLE
Replace CDH 5.9 images with CDH 5.12

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: prestodb/cdh5.12-hive/files/root/setup.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+cd /usr/lib/hive/scripts/metastore/upgrade/mysql/
+echo "GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' WITH GRANT OPTION; FLUSH PRIVILEGES;" | mysql
+echo "CREATE DATABASE metastore; USE metastore; SOURCE hive-schema-1.1.0.mysql.sql;" | mysql
+/usr/bin/mysqladmin -u root password 'root'
+
+killall mysqld
+sleep 10s
+mkdir /var/log/mysql/
+chown mysql:mysql /var/log/mysql/
```
Comment (kokosing) [inline prestodb/cdh5.12-hive/files/root/setup.sh]: no new line at the end

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ dist: trusty
 env:
```
Comment (kokosing) [inline .travis.yml]: it would be nice to know the difference between 5.15

Comment (findepi) [reply .travis.yml]: added note to cmt msg

File: prestodb/cdh5.12-hive/files/root/setup.sh (hunk 1 above)
Comment (findepi) [reply prestodb/cdh5.12-hive/files/root/setup.sh]: verbatim copy

**Pull request merged**